gnome-shell/src/gnome-shell-sass/_drawing.scss

```diff
@@ -115,86 +118,112 @@
 // $tc:   optional text color for colored* types
 // $edge: set to none to not draw the bottom edge or specify a color to not
 //        use the default one
+// $shadow: set to none to not draw the drop shadow or specify a color to not
+//          use the default one
 //
 // possible $t values:
 // normal, hover, active, insensitive, insensitive-active,
 // backdrop, backdrop-active, backdrop-insensitive, backdrop-insensitive-active,
 // osd, osd-hover, osd-active, osd-insensitive, osd-backdrop, undecorated
 //
 
-  $_hilight_color: _button_hilight_color($c);
-  $_button_edge: if($edge == none, none, _widget_edge($edge));
-  $_blank_edge: if($edge == none, none, _widget_edge(transparentize($edge,1)));
-  $_button_shadow: 0 1px 2px transparentize($shadow_color, 0.03);
+  $hilight_color: draw_button_hilight_color($c);
+  $button_edge: if($edge == none, none, draw_widget_edge($edge));
+  $blank_edge: if($edge == none, none, draw_widget_edge(transparentize($edge,1)));
+  $button_shadow: if($shadow == none, none, 0 1px transparentize(black, 0.9));
 
-  @if $t==normal {
-  //
   // normal button
-  //
-
+  @if $t==normal {
     color: $tc;
-    background-color: $c;
-    border-color: $borders_color;
-    box-shadow: $_button_shadow;
-    text-shadow: 0 1px black;
-    icon-shadow: 0 1px black;
+    background-color: lighten($c, 3%);
+    border-color: draw_border_color($c);
+    @include draw_shadows($button_shadow);
+    // box-shadow: 0 1px 1px 0 rgba(0,0,0,0.1);
+    text-shadow: 0 1px $text_shadow_color;
```

Comment:
- background color is lighter now
- border-color was `$borders_color` which had different darken percentages respect this
- shadow is different
- text-shadow was black, while_text_shadow_color is white in light variant and black in dark variant (same below)

Reply:
There can be changes which come from upstream. Maybe this is what happened. 

For the dark theme I've darkened the borders color to look like the gtk borders in the dark theme, otherwise the shell popups are a little bit hard to see